frontend/src/game/Cell.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+    td: {
+        border: "6px solid #222",
+        textAlign: "center",
+        width: "100px",
+        height: "100px",
+        padding: "10px",
+        '&:after': {
+            content: "",
+            display: "block",
+            marginTop: 100,
+        },
+        "&:first-of-type": {
```

Comment:
I'm not sure what this looks like, but I'm sure there is a better way than all these nth-of-type. Maybe we can sit down and you can show me what all this stuff is.